src/browserbehavior/DefaultBrowserBehavior.ts

```diff
@@ -232,7 +223,7 @@ export default class DefaultBrowserBehavior implements BrowserBehavior, Extended
   }
 
   isSimulcastSupported(): boolean {
-    return this.hasChromiumWebRTC() && this.enableUnifiedPlanForChromiumBasedBrowsers;
+    return this.requiresUnifiedPlan() && this.hasChromiumWebRTC();
```

Comment:
Yeah for now we don't need to. But I leave the interface (requiresUnifiedPlan()) here just in case in future we make any changes on how we define requiresUnifiedPlan(), or make logic clearer say if we add support for Safari later.

Reply:
Nvm since we are gonna update other simulcast related codes and docs, I will just go ahead to remove the signature here.